### ISSUE TITLE
Fields list in selects

### ISSUE THREAD

**Issue by grechkin-pogrebnyakov** [jirfag/go-queryset#17]
Now it generates:
```
// One is used to retrieve one result. It returns gorm.ErrRecordNotFound
// if nothing was fetched
func (qs PaymentQuerySet) One(ret *Payment) error {
 return qs.db.First(ret).Error
}
```
If would be awesome if it was this like:
```
// One is used to retrieve one result. It returns gorm.ErrRecordNotFound
// if nothing was fetched
func (qs PaymentQuerySet) One(ret *Payment, fields ...paymentDBSchemaField) error {
 if (len(fields) != 0) {
  qs = qs.w(qs.db.Select(fields...))
 }
 return qs.db.First(ret).Error
}
```

**Comment by ronnie-stratoscale** [comment]
I think that pull request #58 handles this.

**Comment by derElektrobesen** [comment]
Fixed in #58
